integration/test/basic/basic_test.go

```diff
@@ -63,25 +59,57 @@ func TestChartConfigChartsInstalled(t *testing.T) {
 		return
 	}
 
-	guestNamespace := "giantswarm"
 	guestG8sClient := g.G8sClient()
 
 	// Wait for chart configs as they may not have been created yet.
 	err := waitForChartConfigs(guestG8sClient)
 	if err != nil {
 		t.Fatalf("could not get chartconfigs %v", err)
 	}
+}
+
+func TestChartConfigPatchDeploy(t *testing.T) {
```

Comment:
The diff is weird, this is the new test. Some logic from `TestChartConfigChartsInstalled` has been moved to here, since its already tested there anyways.

Reply:
Does that mean there are two different tests in one test package? That is a bit against the idea of the test packages then.

Reply:
Yes I was considering putting it in another package. But since this is closely related I thought its worth the tradeoff to not have another tenant cluster spin up, since cluster-operator e2e are already a pain waiting.

Reply:
Yes the current situation with needing a tenant cluster is painful. Lets keep the test here for now.

This will all change with catalog-operator / app-operator. Once its in place cluster-operator will no longer connect to tenant clusters.